src/store/actions.js

```diff
@@ -7,13 +7,27 @@ import {
     START_RETURN__REQUEST,
     START_RETURN__SUCCESS,
     START_RETURN__FAILURE,
+    RETURN_EXCHANGE_DETAILS,
     GET_RETURN_STATUS__REQUEST,
     GET_RETURN_STATUS__SUCCESS,
     GET_RETURN_STATUS__FAILURE,
-    GO_HOME,
+    GO_HOME
 } from "./actionTypes"
 
 
+export const returnExchangeDetails = (quantityToReturn, quantityToExchange, keepRefundExchange, refundReason, exchangeReason, exchangeToSize, itemComments) => ({
```

Comment:
Is this action to be run for each item, or for the entire return/exchange object?